week-2/F-logical-operators/exercise.js

```diff
@@ -11,14 +11,14 @@ var cssLevel = 4;
 
 // Finish the statement to check whether HTML, CSS knowledge are above 5
 // (hint: use the comparison operator from before)
-var htmlLevelAbove5;
-var cssLevelAbove5;
+var htmlLevelAbove5 = 5 < 8;
+var cssLevelAbove5 = 5 < 4;
 
 // Finish the next two statement
 // Use the previous variables and logical operators
 // Do not "hardcode" the answers
-var cssAndHtmlAbove5;
-var cssOrHtmlAbove5;
+var cssAndHtmlAbove5 = 5 > 12;
+var cssOrHtmlAbove5 = 5 <= 8;
 
```

Comment:
Try to understand the task, you're given two variables htmlLevel and cssLevel use them to do your checks